mesh_tools/create_SCRIP_files/create_SCRIP_file_from_CISM_mesh.py

```diff
@@ -16,60 +17,70 @@
-projections['gis-bamber'] = pyproj.Proj('+proj=stere +lat_ts=71.0 +lat_0=90 +lon_0=321.0 +k_0=1.0 +x_0=800000.0 +y_0=3400000.0 +ellps=WGS84')  # This version ignores the vertical datum shift, which should be a very small error for horizontal-only positions
+#projections['gis-bamber'] = '+proj=stere +lat_ts=71.0 +lat_0=90 +lon_0=321.0 +k_0=1.0 +x_0=800000.0 +y_0=3400000.0 +ellps=WGS84 +geoidgrids=./egm08_25.gtx'
+projections['gis-bamber'] = '+proj=stere +lat_ts=71.0 +lat_0=90 +lon_0=321.0 +k_0=1.0 +x_0=800000.0 +y_0=3400000.0 +ellps=WGS84' # This version ignores the vertical datum shift, which should be a very small error for horizontal-only positions
 
 # GIMP projection: This is also polar stereographic but with different standard parallel and using the WGS84 ellipsoid.
-projections['gis-gimp'] = pyproj.Proj('+proj=stere +lat_ts=70.0 +lat_0=90 +lon_0=315.0 +k_0=1.0 +x_0=0.0 +y_0=0.0 +ellps=WGS84')
+projections['gis-gimp'] = '+proj=stere +lat_ts=70.0 +lat_0=90 +lon_0=315.0 +k_0=1.0 +x_0=0.0 +y_0=0.0 +ellps=WGS84'
 
 # BEDMAP2 projection
-projections['ais-bedmap2'] = pyproj.Proj('+proj=stere +lat_ts=-71.0 +lat_0=-90 +lon_0=0.0 +k_0=1.0 +x_0=0.0 +y_0=0.0 +ellps=WGS84')  # Note: BEDMAP2 elevations use EIGEN-GL04C geoid
+projections['ais-bedmap2'] = '+proj=stere +lat_ts=-71.0 +lat_0=-90 +lon_0=0.0 +k_0=1.0 +x_0=0.0 +y_0=0.0 +ellps=WGS84' # Note: BEDMAP2 elevations use EIGEN-GL04C geoid
 
 # Standard Lat/Long
-projections['latlon'] = pyproj.Proj(proj='latlong', datum='WGS84')
-# ===================================
+projections['latlon'] = '+proj=longlat +ellps=WGS84'
 
 
 
-print "== Gathering information.  (Invoke with --help for more details. All arguments are optional)"
+print ("== Gathering information.  (Invoke with --help for more details. All arguments are optional)")
 parser = OptionParser()
 parser.description = "This script takes an MPAS grid file and generates a SCRIP grid file."
-parser.add_option("-c", "--cism", dest="cismFile", help="CISM grid file name used as input.", default="cism.nc", metavar="FILENAME")
+parser.add_option("-i", "--input", dest="inputFile", help="input grid file name used as input.", default="input.nc", metavar="FILENAME")
 parser.add_option("-s", "--scrip", dest="scripFile", help="SCRIP grid file to output.", default="scrip.nc", metavar="FILENAME")
-parser.add_option("-p", "--proj", dest="projection", help="projection used by the CISM file. Valid options are:"  + str(projections.keys()), metavar="PROJ")
+parser.add_option("-p", "--proj", dest="projection", help="projection used by the input data file. Valid options are:"  + str(projections.keys()), metavar="PROJ")
+parser.add_option("-r", "--rank", dest="gridRank", help="desired rank of the output SCRIP grid data")
+parser.add_option("--plot", dest="plot", help="destination grid points are plotted if 'True'. Default is 'False'", default=False)
```

Comment:
There's a slightly cleaner way to handle this:
https://docs.python.org/3/library/optparse.html#handling-boolean-flag-options

```suggestion
parser.add_option("--plot", dest="plot', action="store_true", help="If this flag is used, destination grid points are plotted")
```